File: PharmaDicBackEnd/PharmaDicBackEnd.ApiService/Models/DiseaseMedicine.cs
using System;
using System.Collections.Generic;

namespace PharmaDicBackEnd.ApiService.Models;

public partial class DiseaseMedicine
{
    public int DiseaseId { get; set; }

    public int MedicineId { get; set; }

    public string? Note { get; set; }

    public virtual Disease Disease { get; set; } = null!;

    public virtual Medicine Medicine { get; set; } = null!;
}

File: PharmaDicMobile/app/src/main/java/com/example/pharmadicmobile/models/DiseaseMedicine.java
package com.example.pharmadicmobile.models;

import java.io.Serializable;

public class DiseaseMedicine implements Serializable {
    private int diseaseId;
    private int medicineId;
    private String note;
    private Disease disease;
    private Medicine medicine;

    public DiseaseMedicine() {}

    public int getDiseaseId() { return diseaseId; }
    public void setDiseaseId(int diseaseId) { this.diseaseId = diseaseId; }

    public int getMedicineId() { return medicineId; }
    public void setMedicineId(int medicineId) { this.medicineId = medicineId; }

    public String getNote() { return note; }
    public void setNote(String note) { this.note = note; }

    public Disease getDisease() { return disease; }
    public void setDisease(Disease disease) { this.disease = disease; }

    public Medicine getMedicine() { return medicine; }
    public void setMedicine(Medicine medicine) { this.medicine = medicine; }
}
